src/pages/JobView/JobView.tsx

```diff
@@ -29,7 +29,7 @@ const JobView = () => {
   }, [getJob, params.id, id, job])
 
   return (
-    <Box>
+    <>
```

Comment:
This doesn't appear to even need to be nested. It could just return the `Grid` (though it shouldn't be a `Grid`, that's a separate issue...)